p/scripts/main.js

```diff
@@ -1080,6 +1081,10 @@ function init_shortcuts() {
 			ev.preventDefault();
 			return;
 		}
+		if (ev.key === '?' && ev.shiftKey) {
```

Comment:
On some keyboard layouts, <kbd>Shift</kbd> must always be pressed to produce a <kbd>?</kbd>
We could consider removing the `ev.shiftKey` test

Reply:
What do you think @Frenzie ?

Reply:
The shift test feels redundant or perhaps even negative for keyboard layouts that don't require pressing shift to produce a question mark.

Reply:
https://github.com/FreshRSS/FreshRSS/pull/6981/commits/ee30c8a291125d39c8a047d84c8e21d7041b792a